locale/en/translation.json

```diff
@@ -117,5 +117,22 @@
   "fab": {
     "scrollToTop": "Scroll to top",
     "goToFeedback": "Was this page helpful?"
+  },
+  "versionFilter": {
+    "yes": "Yes",
+    "no": "No",
+    "filterModeVersion": "Version",
+    "filterModeDiff": "Diff",
+    "diffModeOlderVersion": "Older Version",
+    "diffModeNewerVersion": "Newer Version",
+    "involvedInitVersion": "From the beginning",
+    "involvedVersion": "New in {{version}}",
```

Comment:
IMO, "introduced" would be more clear than "involved".

Reply:
OK

Reply:
Done